### ISSUE TITLE
[New Feature] Runner: Do not scroll up summary

### ISSUE THREAD

**Issue by tho-gru-38** [usebruno/bruno#3270]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

While a collections runs the runners shows a summary at the top:
![Runner-01](https://github.com/user-attachments/assets/273bda5a-5ed5-492b-a48f-203c325587b4)

During execution the summary scrolls up. In the end the runner looks like this:
![Runner-02](https://github.com/user-attachments/assets/15ffea42-b59e-4f6e-8364-e01a0b86865d)

It is possible to see the summary always during the execution of a collection.

The auto scroll is very useful. I just want to see the summary while the runner executes the collection.

### Mockups or Images of the feature

The runner view should always look like this:
![Runner-mock](https://github.com/user-attachments/assets/f8c105c0-0810-4c6c-b3b9-13375bc6ddea)

**Comment by ganesh-bruno** [comment]
This will be a valuable addition. Right now, we're focused on fixing essential onboarding issues.
We value quality improvements and understand their importance, but we need a bit more time before we can get to them.

**Comment by nikischin** [comment]
I agree this should be implemented and could be easily done I guess. Also whenever I click on some result I wouldn't want the scroll to jump to the end of the page but instead remain at the current position.

This seems like related to some wrong dependency in the useEffect in `<RunnerResults />`. This is quite frustrating if you have many runner items.

**Comment by gh47110815** [comment]
YES - Please Do not scroll up summary OR place summary and buttons for "run again" ... together
(I also just wanted to open an issue for that)

Regarding the runner there are more enhancements I would like to see...
1. if you click on a request inside the collections tree to the left, runners output jumps away from the last position. That should be fixed
2. it would be nice to be able to directly jump from the runners window to the corresponding request. 
3. In brunos "best friend" the user was able to click on "all", "failed", "succeeded" tests. So it was possible to only monitor or view e.g. failing tests. that was a nice feature I used very often. 
4. if you are starting the runner and no Environment was selected .... this should be checked and e.g. prompted: "please choose an environment" - or user should be able to set a default Environment. Running a collection recursively with environment variables like the URL doesn't make sense ;-)

**Comment by tho-gru-38** [comment]
HI,

I think this issue is implemented with version 2.9.0.

Kind Regards
